stubs/redis/redis/__init__.pyi

```diff
@@ -59,3 +59,6 @@ RedisError = exceptions.RedisError
 ResponseError = exceptions.ResponseError
 TimeoutError = exceptions.TimeoutError
 WatchError = exceptions.WatchError
+
+__version__: str
+VERSION: tuple[int | str]
```

Comment:
```suggestion
VERSION: tuple[int | str, ...]
```

Otherwise you're saying it will always be a tuple of length 1

Reply:
Of course